pkg/web/router.go

```diff
@@ -148,6 +154,7 @@ func NewRouter(config *RouterConfig, wpool pool.WorkerPoolInterface) http.Handle
 	rulesAPI.RegisterEndpoints(router)
 	rolesAPI.RegisterEndpoints(router)
 	groupsAPI.RegisterEndpoints(router)
+	uiAPI.RegisterEndpoints(router)
```

Comment:
this won't fly imho, as all the middlewares for authorisation are hooked already, UI shouldn't be subject to those

Reply:
That's right, the UI should be delivered to any unauthorized requester.

Reply:
Good point, didn't notice the middleware was there.

Interesting thing I learned about chi router (even though it is not documented):
Using `With` AFTER you register a handler will not affect that handler, so this gives us the behavior we want:
```
	uiAPI.RegisterEndpoints(router)
	router := router.With(...)
        // Register all other handlers
	statusAPI.RegisterEndpoints(apiRouter)
	...
```

Using `With` last causes a panic:
 ```
        // Register all handlers
	uiAPI.RegisterEndpoints(router)
	statusAPI.RegisterEndpoints(apiRouter)
	...
	router := router.With(...)
```

I went with the `Mount` approach and changed the paths on all the handlers, from an API perspective nothing should have changed.